osu.Game/Screens/Edit/Compose/Components/SelectionBox.cs

```diff
@@ -92,21 +95,32 @@ public bool CanScaleY
             }
         }
 
+        private string text;
+
+        public string Text
+        {
+            get => text;
+            set
+            {
+                if (value == text)
+                    return;
+
+                text = value;
+                if (selectionDetailsText != null)
+                    selectionDetailsText.Text = value;
+            }
+        }
+
         private Container dragHandles;
         private FillFlowContainer buttons;
 
-        public const float BORDER_RADIUS = 3;
+        private OsuSpriteText selectionDetailsText;
 
         [Resolved]
         private OsuColour colours { get; set; }
 
         [BackgroundDependencyLoader]
-        private void load()
-        {
-            RelativeSizeAxes = Axes.Both;
-
-            recreate();
-        }
+        private void load() => recreate();
```

Comment:
iirc BDLs with expression bodies were not wanted, statement bodies look better for these cases.
```suggestion
        private void load() 
        {
             recreate();
        }
```

Reply:
this comment is irrelevant. a method is a method is a method. and there are even [existing examples](https://github.com/ppy/osu/blob/d8d2c731ee42db1ecc7e1a6c8cb3d5f1dad8b943/osu.Game/Screens/Play/HUD/DefaultComboCounter.cs#L26-L27) of expression-bodied BDLs.

Reply:
this comment is not irrelevant. I was mistakingly searching parameterless BDL so couldn't catch one, but I'm 99.99% sure I was suggested to always use statement bodies for BDL, could be my memory not serving me.